### ISSUE TITLE
Attrs for XML does not work.

### ISSUE THREAD

**Issue by jovicailievski** [ionic-team/trapeze#155]
This does not work: 
```
   xml:
      - file: app/src/main/res/values/styles.xml
        target: resources
        attrs:
          test: test
```

But for the manifest it works:
```
      - file: AndroidManifest.xml
        target: manifest
        attrs:
          test: test
```

Here in xml.js I modified the function to print the args.
```
    setAttrs(target, attrs) {
        console.log("target:",target, "attrs:", attrs)
        var _a, _b;
        if (!this.doc) {
            return;
        }
        logger_1.Logger.v('xml', 'setAttrs', `at ${this.path} - ${target}`);
        const nodes = (_b = (_a = this.select) === null || _a === void 0 ? void 0 : _a.call(this, target, this.doc)) !== null && _b !== void 0 ? _b : [];
        nodes.forEach((n) => {
            Object.keys(attrs).forEach(attr => {
                n.setAttribute(attr, attrs[attr]);
            });
        });
        this.vfs.set(this.path, this);
    }
```
Output is: _target: resources attrs: undefined_

Error is: 
TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at C:\Users\exfled\Desktop\Projects\tcgbulkapp\node_modules\@trapezedev\project\dist\xml.js:219:20
    at Array.forEach (<anonymous>)
    at XmlFile.setAttrs (C:\Users\exfled\Desktop\Projects\tcgbulkapp\node_modules\@trapezedev\project\dist\xml.js:218:15)
    at execute (C:\Users\exfled\Desktop\Projects\tcgbulkapp\node_modules\@trapezedev\configure\dist\operations\android\xml.js:30:27)
    at async executeOperations (C:\Users\exfled\Desktop\Projects\tcgbulkapp\node_modules\@trapezedev\configure\dist\tasks\run.js:52:10)
    at async runCommand (C:\Users\exfled\Desktop\Projects\tcgbulkapp\node_modules\@trapezedev\configure\dist\tasks\run.js:31:9)
    at async C:\Users\exfled\Desktop\Projects\tcgbulkapp\node_modules\@trapezedev\configure\dist\index.js:56:13
    at async Command.<anonymous> (C:\Users\exfled\Desktop\Projects\tcgbulkapp\node_modules\@trapezedev\configure\dist\util\cli.js:31:13)

Tested for nested tags, it doesn't work too.

**Comment by mlynch** [comment]
My bad, fixed and added tests. Will get this out shortly in the next release

**Comment by mlynch** [comment]
Fixed in 7.0.5
